android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationOperation.kt

```diff
@@ -23,7 +23,7 @@ enum class TranslationOperation {
      * the languages supported for translating both "to" and "from" with their BCP-47 language tag
      * and localized name.
      */
-    FETCH_LANGUAGES,
+    FETCH_SUPPORTED_LANGUAGES,
```

Comment:
Changed because I think this might need some disambiguation from the future addition of `FETCH_DOWNLOADED_LANGUAGES`.